opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -19,13 +19,16 @@
 
 from __future__ import annotations
 
+import inspect
 import logging
 import logging.config
 import os
 from abc import ABC, abstractmethod
 from os import environ
-from typing import Any, Callable, Mapping, Sequence, Type, Union
+from typing import Any, Callable, Mapping, Optional, Sequence, Type, Union
 
+from grpc import ChannelCredentials  # pylint: disable=import-error
```

Comment:
Could this be an issue if the SDK is running in an environment without grpcio installed? I know the operator auto-instrumentation layer for Python [pre-installs the HTTP exporters](https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/python/requirements.txt#L4-L6) and not the grpc exporters to avoid extension issues.